packages/instantsearch.js/src/lib/utils/addWidgetId.ts

```diff
@@ -0,0 +1,11 @@
+import type { Widget } from '../../types';
+
+let id = 0;
+
+export function addWidgetId(widget: Widget) {
```

Comment:
no big deal, but this same code is in the other connector PR as well and actually should be in the base branch

Reply:
Yes but I guess we can just handle conflicts, this shouldn't be too hard.